Apps/Admin/Client/Store/Analytics/AnalyticsActions.cs

```diff
@@ -0,0 +1,155 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Admin.Client.Store.Analytics;
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Net.Http;
+
+/// <summary>
+/// Static class that implements all actions for the feature.
+/// </summary>
+[SuppressMessage("Design", "CA1034:Nested types should not be visible", Justification = "Team decision")]
+public static class AnalyticsActions
+{
+    /// <summary>
+    /// The action representing the initiation of user profiles action load.
```

Comment:
Nit: "user profiles action load" should be "user profiles load action", but since action is mentioned at the start of the comment, it can be simplified to "user profiles load".  (Applies throughout file.)